fulcio.crt.pem

```diff
@@ -0,0 +1,13 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Accidental check-in, or is this a test asset? If so, we should probably put it under `test/assets`

Reply:
Oh, sorry - I see that it's a root cert now! In that case, it should probably under `sigstore` and also recorded in `setup.py` or a `MANIFEST.in` file so that it correctly shows up in source and binary distributions.